godot_project/editor/controls/menu_bar/msep_popup_menu.gd

```diff
@@ -64,15 +64,5 @@ func _update_menu() -> void:
 	assert(false, "Implement this function in your specialized class")
 
 
-func _on_forward_id_pressed(in_id: int) -> void:
-	var workspace_context: WorkspaceContext = MolecularEditorContext.get_current_workspace_context() as WorkspaceContext
-	if is_instance_valid(workspace_context):
-		if workspace_context.get_editor_viewport().has_exclusive_input_consumer():
-			# block menu activations (assumed from keyboard shortcuts)
-			# when viewport has an exclusive consumer
-			return
-	_on_id_pressed(in_id)
-
```

Comment:
This was never called, or rather, it jumped to `_on_id_pressed` directly all the time.
Because the input is set as handled in viewport_input.gd:49, the `id_pressed` event is not triggered when there is an exclusive input handler.